### PR TITLE
Update phab and node

### PR DESCRIPTION
**Pull request by mikran**
pipelines are failing on node version

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-FROM thecodingmachine/php:7.4-v4-cli-node14
+FROM thecodingmachine/php:7.4.27-v4-apache-node16
```
Comment (stmh) [inline Dockerfile]: I wonder why you switched to the apache-version instead of the cli one?

**Comment by stmh** [comment]
hi @mikran 

thanks for the changes. Unfortunately docker hub does not build the images automatically on repo changes anymore, so we need to add some github-actions or build or build and push the image manually. Do you want to volunteer und add the missing github actions? (We added them for Drupal-docker recently, might be a good inspiration)

File: Dockerfile (hunk 1 above)
Comment (mikran) [reply Dockerfile]: Thanks for spotting it! that was not intended

**Comment by mikran** [comment]
> hi @mikran
> 
> thanks for the changes. Unfortunately docker hub does not build the images automatically on repo changes anymore, so we need to add some github-actions or build or build and push the image manually. Do you want to volunteer und add the missing github actions? (We added them for Drupal-docker recently, might be a good inspiration)

yep, I can do that

**Comment by mikran** [comment]
Hi @stmh 

This was far simpler than what was done with drupal-docker, maybe those actions can be updated to use https://github.com/marketplace/actions/build-and-push-docker-images too

To make this work repo settings need secrets 

```
          username: ${{ secrets.DOCKERHUB_USERNAME }}
          password: ${{ secrets.DOCKERHUB_TOKEN }}
```

**Review by stmh — COMMENTED** [review]
looks good to me, added the secrets, lets see what is happening :)

**Pull request merged**